README.md

```diff
@@ -6,10 +6,11 @@ This project tries to reproduce Ubuntu’s `command-not-found` for Homebrew user
 on OSX.
 
 On Ubuntu, when you try to use a command that doesn’t exist locally but is
-available through a package, Bash will suggest you a command to install it.
+available through a package, your shell will suggest you a command to install it.
+
```

Comment:
Please don’t change this, it’s still true because it’s talking about Ubuntu, not OS X.